### ISSUE TITLE
Submit to IMCIC 2022 by September 15

### ISSUE THREAD

**Issue by shatayu** [opendigital/RL-collective-action#9]
**TL;DR: Submit article by September 15, decision by October 27**

Dear Sabine Brunswicker,

We are sending you this email to invite you to contribute to any of the topics suggested in the 13th International Multi-Conference on Complexity, Informatics and Cybernetics: IMCIC 2022, which will be held on March 8 - 11, 2022, in Orlando, Florida, USA.

You can submit an article at: https://www.iiis-2022conf.org/imcic

This conference has always contemplated the two modalities of participation: face to face and virtual. However, given the current Coronavirus situation and travel restrictions, at this moment we are only accepting submissions for virtual participation. If the situation changes and we feel the safety of our participants would not be at risk, we will start accepting submissions for face-to-face participation, and those authors who submitted their article for virtual participation will have the ability to change their participation modality to face-to-face, if they wish.

The Proceedings of IMCIC and ICSIT/ICETI have been indexed by Elsevier's SCOPUS since 2010. The 2022 proceedings will also be sent to Elsevier's SCOPUS.

The conference will also be jointly held with:
The 13th International Conference on Education, Training and Informatics: ICETI 2022
The 13th International Conference on Society and Information Technologies: ICSIT 2022
More details regarding these and other collocated events and special tracks can be found at: https://www.iiis-2022conf.org/cfp-spring2022.asp

The deadlines for this CFP are as follows:
September 15th, 2021: Article submissions
September 15th, 2021: Invited/Special session proposals
October 27th, 2021: Notifications of acceptance
December 2nd, 2021: Uploading of camera-ready or final version

Both synchronous and asynchronous virtual presentations are possible. Therefore, the authors who participate virtually in the conference have the option of:
Submitting a slide show and/or a pre-recorded video of their presentation (asynchronous presentation) or
Making their presentation via Zoom (synchronous presentation)

Additionally, each paper to be presented at the event, virtually or in-person, may be read and commented on, via electronic forums, and evaluated by those participants who have registered for the conference.

You also have the option to contribute as a member of the Organizing Committee by accepting our invitation for organizing an invited or special session, where you would have the responsibility of promoting your session and reviewing the articles submitted to you. You should appoint your own reviewers to support you. Effective organizers will be included as co-editors of the volume of the proceedings in which the papers of the session(s) that they organized have been included. If you accept this invitation, please submit your proposal through the conference web site.

Keynote speakers will be selected from registered authors whose submissions are among the best according to their external reviewers. The identification of the best submissions will be based on the quantitative and qualitative evaluation of the reviewers and the percentage of reviewers recommending the acceptance of the submission as related to all the reviewers who made their respective reviews. For this reason, we recommend you to submit your paper early.

Registered authors of accepted submissions will be invited to make an additional presentation with no additional charge , if such a presentation is oriented to inter-disciplinary communication. These authors will also have the option of writing an invited paper associated with their respective invited presentation.

Furthermore, the best papers presented at the conference, as selected by the audiences of each respective session, will also be published in the Journal of Systemics, Cybernetics, and Informatics (JCSI). JSCI is indexed in EBSCO, Cabell, DOAJ, and Google Scholar and is listed in Cabell Directory of Publishing Opportunities and in Ulrich's Periodical Directory.

For the past 26 years, professors, students, and researchers from all over the world have come together at our conferences to present research and engage in interdisciplinary discussion. It is this interdisciplinary focus that allows our attendees to discover new viewpoints they otherwise would not have discovered in a uni-disciplinary conference. These new viewpoints become intellectual seeds which, as they slowly bloom over time, expand their understanding of their own respective disciplines.

Many of our participants look forward to our conferences every year as they allow them to publish their papers, share their life's work, create a network of influential and like-minded colleagues, and expand their ever-growing understanding of their respective disciplines, as well as learn about new disciplines. Some testimonials can be read clicking on the button "TESTIMONIALS" placed at the top of the left column of each of the conference web sites.

We hope to continue creating and expanding inter-disciplinary networks while facilitating the ever-growing expansion of each individual discipline represented at our conferences. We know this is only possible with contributions from people such as yourself. It is for this reason that we would greatly appreciate your contribution to the conference.

You are also welcome to forward this message to any appropriate colleagues and/or groups whose contributions would further promote the interdisciplinary growth of all the various disciplines represented in our conferences.

Thank you for your time, and we look forward to your contribution.

Respectfully,

IMCIC 2022 Organizing Committee
